source/dagger/gameplay/ping_pong/player_scores.h

```diff
@@ -28,5 +28,12 @@ namespace ping_pong
             s_FieldHeight = height;
             s_TileSize = tileSize;
         }
+
+		float GetRandomSpeed()
+		{
+			float x_speed = rand() % 10 - 5.0f;
+			x_speed = x_speed ? x_speed : x_speed + 5.0f;
```

Comment:
you are trying to compare with 0 here?
it's bad practice to compare float with 0 like that. You need to use > < operators instead.